README.md

```diff
@@ -20,11 +20,11 @@ Three types of applications are available: for the **desktop** (Windows, macOS a
 
 Operating System | Download
 ---|---
-Windows (32 and 64-bit) | <a href='https://github.com/laurent22/joplin/releases/download/v1.5.11/Joplin-Setup-1.5.11.exe'><img alt='Get it on Windows' width="134px" src='https://joplinapp.org/images/BadgeWindows.png'/></a>
-macOS | <a href='https://github.com/laurent22/joplin/releases/download/v1.5.11/Joplin-1.5.11.dmg'><img alt='Get it on macOS' width="134px" src='https://joplinapp.org/images/BadgeMacOS.png'/></a>
-Linux | <a href='https://github.com/laurent22/joplin/releases/download/v1.5.11/Joplin-1.5.11.AppImage'><img alt='Get it on Linux' width="134px" src='https://joplinapp.org/images/BadgeLinux.png'/></a>
+Windows (32 and 64-bit) | <a href='https://github.com/laurent22/joplin/releases/download/v1.5.14/Joplin-Setup-1.5.14.exe'><img alt='Get it on Windows' width="134px" src='https://joplinapp.org/images/BadgeWindows.png'/></a>
+macOS | <a href='https://github.com/laurent22/joplin/releases/download/v1.5.14/Joplin-1.5.14.dmg'><img alt='Get it on macOS' width="134px" src='https://joplinapp.org/images/BadgeMacOS.png'/></a>
+Linux (x86) | <a href='https://github.com/laurent22/joplin/releases/download/v1.5.14/Joplin-1.5.14.AppImage'><img alt='Get it on Linux' width="134px" src='https://joplinapp.org/images/BadgeLinux.png'/></a>
 
-**On Windows**, you may also use the <a href='https://github.com/laurent22/joplin/releases/download/v1.5.11/JoplinPortable.exe'>Portable version</a>. The [portable application](https://en.wikipedia.org/wiki/Portable_application) allows installing the software on a portable device such as a USB key. Simply copy the file JoplinPortable.exe in any directory on that USB key ; the application will then create a directory called "JoplinProfile" next to the executable file.
+**On Windows**, you may also use the <a href='https://github.com/laurent22/joplin/releases/download/v1.5.14/JoplinPortable.exe'>Portable version</a>. The [portable application](https://en.wikipedia.org/wiki/Portable_application) allows installing the software on a portable device such as a USB key. Simply copy the file JoplinPortable.exe in any directory on that USB key ; the application will then create a directory called "JoplinProfile" next to the executable file.
```

Comment:
Is this not an issue? If you click the Android play store link it takes you to the v.1.5.1 updated version but the apk link takes to you an older 1.4.11. Similarly on the desktop links it takes you to the download of the older 1.5.11 version then if you select 'check for updates' you are immediately prompted for 1.5.14 download.
I included x86 because I noticed there have been a number of requests and comments that people have been unable to make it work on Raspberry Pi, Pinebooks etc. but can remove it if required.

Reply:
Version numbers are updated automatically. Yes please remove the x86 mention, I'm trying to keep this download table simple.

Reply:
What is updating them automatically, has this broken? As I mentioned, if you look at the readme (+ joplinapp website), the manual download links all reference an old version where the versions downloaded from repositories (linux command line, play store, app store) all link to the newer version.

Reply:
No, someone has to run a sscript in the `pacakages/tools` dir. However, we only run this, when there's an official release for desktop.

Reply:
Then has this not been done? I'll revert the changes now I know that it is auto, my assumption was that it wasn't because of the mismatch of versions between repos and links. 1.5.14 and 1.5.1 are the latest "proper" released versions are they not? As mentioned this is not what the current document links to. 
![image](https://user-images.githubusercontent.com/58074586/104107177-4d54ee00-52b2-11eb-919c-8f83d025b7d9.png)

Reply:
Apparently we forgot. So after we merge the PR, we update it. That is what Laurent was saying, also in rwference to the `terminal.md` file, when it comes to the text for the joplin command (options and arguments).

Reply:
Right, I get you. The terminal.md ones aren't automatic at the moment though are they, so those would have to be done (as Laurent said) in a future PR - I very much doubt I can fathom out how to do it within this one.

Reply:
They might be. I don't know. I would have to read the scripts in tools. But yeah, this is certainly not part of this PR.